src/rgw/rgw-restore-bucket-index

```diff
@@ -34,6 +34,18 @@ export obj_list=/tmp/rgwrbi-object-list.$$
 export zone_info=/tmp/rgwrbi-zone-info.$$
 export clean_temps=1
 
+# number of seconds for a bucket index pending op to be completed via
+# dir_suggest mechanism
+pending_op_secs=120
```

Comment:
`bucket check --fix` uses a trick to temporarily reduce the [tag timeout](https://github.com/ceph/ceph/blob/b0615fd/src/rgw/driver/rados/rgw_bucket.cc#L432-L433). that might help to speed things up?

Reply:
The challenge is that process is wholly run within a single invocation of radosgw-admin. At the moment, a shell script runs radosgw-admin twice, once for re-index, once for unordered listing. So either we'd have to move everything in radosgw-admin or perhaps allow an option for unordered listing to shorten pending_op_seconds.